Lib/gzip.py

```diff
@@ -17,6 +17,15 @@
 
 READ, WRITE = 1, 2
 
+# OS field values as specified in RFC 1952
+OS_CODES = {
```

Comment:
If values such as `OS_CODES` does not intend to expose as the public constants.
 I'd like to recommend to change `OS_CODES` into `_OS_CODES` also for  `OS_UNIX`,  `OS_NTFS`.

what do you think @serhiy-storchaka

Reply:
I agree. There is no need to expose them since they are specific to gzip header only.

Reply:
Fixed in 9db21eb82f738193d22add4168bac62279de54e7